baseplate/server/runtime_monitor.py

```diff
@@ -108,7 +124,8 @@ def _report_runtime_metrics_periodically(metrics_client, reporters):
 
 
 def start(server_config, application, pool):
-    if not hasattr(application, "baseplate") or not application.baseplate._metrics_client:
+    baseplate = getattr(application, "baseplate", None)
+    if not baseplate or not baseplate._metrics_client:
```

Comment:
Unrelated change, or am I missing something?

Reply:
I wanted to keep the `baseplate` object around so I could use it when building the reporter thing down below. Before it was just ephemerally present in this check to get the metrics off of it.